lib/DataIntegrityProof.js

```diff
@@ -292,6 +292,17 @@ export class DataIntegrityProof extends LinkedDataProof {
       if(!verified) {
         throw new Error('Invalid signature.');
       }
+      if(proof.created) {
+        if(!util.isW3cDate(proof.created)) {
+          throw new Error('invalid datetime format');
+        }
+        const now = new Date();
+        const created = util.convertTimeStamp(proof.created);
+        const deltaCreated = now - created;
```

Comment:
We also need `maxClockSkew` or this won't work properly in real settings. So we're going to need to figure a few things out here.

Reply:
`maxClockSkew` is now an option passed to `DataIntegrityProof` and defaults to 300 seconds: https://github.com/digitalbazaar/data-integrity/pull/34/commits/df8f6fdbe027d220426e8bc4b390939f1fca6e1e